collections/src/main/scala/strawman/collection/SortedMap.scala

```diff
@@ -59,6 +59,18 @@ trait SortedMapOps[K, +V, +CC[X, Y] <: Map[X, Y] with SortedMapOps[X, Y, CC, _],
   def firstKey: K = head._1
   def lastKey: K = last._1
 
+  /** Find the element with smallest key larger than or equal to a given key.
+    * @param key The given key.
+    * @return `None` if there is no such node.
+    */
+  def minAfter(key: K) = from(key).headOption
```

Comment:
Could you add the return type explicitly?

Reply:
Also, I’d be tempted to not have a default implementation if we always override it in leaf classes.